packages/clay-dropdown/src/ClayDropdownBase.soy

```diff
@@ -364,6 +364,9 @@
 
 	<button {$buttonAttributes}>
```

Comment:
@georgel-pop-lr, I see we're adding an `onClick` but I don't think it's needed. It's been a long time since I looked at .soy templates, but this `{$buttonAttributes}` is probably what you want to use. Have you tried passing the "onClick" through there?

Reply:
I'm checking how to use `defaultEventHandler` on 7.2 but without any success: https://liferay.slack.com/archives/C3JBR21HA/p1637858723151500

I see on master is using more or less same logic, but with `propsTransformer`.

Reply:
Hey @georgel-pop-lr I passed some information through Slack, I think this is no longer necessary because the way we handle events in v2 is common for all components and maybe we don't need to explicitly add the `onClick` property.

All events emitted by the component up to the Metal.js lifecycle can be listened for using the `events` property when using `Soy` directly and `defaultEventHandler` in 2.x for taglibs, ie when it has an event being emitted as a `click` internally the Clay Component will invoke the equivalent method on the instance of `defaultEventHandler`.

Reply:
Hi @matuzalemsteles,

Yes, I'm trying out and see if is working, the problem here is also to send a param from JSP to the JS event of defaultEventHandler, for the moment not sure how that it can be done. 

Regards.

Reply:
Sorry @georgel-pop-lr, this change has nothing to do with the "onClick" event, sorry about that.
As we spoke together on Slack, I'd rather have this change:

```soy
{if $title}
    <span class="sr-only">{$title}</span>
{/if}
``` 

in the ["content"](https://github.com/liferay/clay/blob/3b15bc306bf342f9eb5778fc618227d80b7206c2/packages/clay-button/src/ClayButton.soy#L104-L130) template. You can apply this rule for all changes in this commit. Thanks.